.github/workflows/plugin-release.yml

```diff
@@ -0,0 +1,48 @@
+name: Plugin-Release
+
+on: pull_request
```

Comment:
This will change to `on: workflow_dispatch`, keeping it on `pull_request` temporarily for the CI to trigger the jobs